src/UrlSearchParams.ts

```diff
@@ -115,19 +118,20 @@ export default class UrlSearchParams {
 	 * @param key The key to return the first value for
 	 * @return The first string value for the key
 	 */
-	get(key: string): string {
+	get(key: string): string | null | undefined {
```

Comment:
Perhaps that is more fluent for the API, however we were already returning `null` then `this.has(key)` is `false` - I can simply convert to use `undefined` in the instance?